app/views/components/_chart.html.erb

```diff
@@ -12,7 +12,7 @@
   if percent_metric
     maximum_y = 100
   elsif rows.any?
-    row_maximums = rows.map { |row| row[:values].max }
+    row_maximums = rows.map { |row| row[:values].map{ |i| i || 0 }.max }
```

Comment:
Are there tests we can add for this? (Side-note for the future, this is out of scope and should be a separate PR)

Reply:
I see this is addressed #238